commands/general/help.js

```diff
@@ -1,55 +1,98 @@
 const Command = require(`${process.cwd()}/base/Command.js`);
-
+const {MessageEmbed} = require("discord.js");
```

Comment:
make it `{ MessageEmbed }`

Reply:
Ditto

Reply:
Whoops, missed that one, that was a fix.